website/docs/embedded_components.md

```diff
@@ -10,5 +10,6 @@ JBrowse 2 is a pluggable open-source platform for visualizing, integrating, and
 
 This component consists of a single JBrowse 2 linear view.
 
-- [@jbrowse/linear-genome-view](https://www.npmjs.com/package/@jbrowse/react-linear-genome-view) - linear genome view React component on NPM
+- [@jbrowse/react-linear-genome-view](https://www.npmjs.com/package/@jbrowse/react-linear-genome-view) - linear genome view React component on NPM
+- [@jbrowse/react-circular-genome-view](https://www.npmjs.com/package/@jbrowse/react-circular-genome-view) - circular genome view React component on NPM
```

Comment:
Maybe this should go under its own new heading, so it's not under the LGV heading.